src/rules/property-no-unknown/README.md

```diff
@@ -85,3 +87,20 @@ The following patterns are *not* considered warnings:
   custom: 10px;
 }
 ```
+### `checkPrefixed: true | false` (default: `false`)
+
+This option forces to check vendor prefixed properties.
+
+For example with `true` the following patterns are considered  warnings:
```

Comment:
Only these specific properties are considered warnings: `-moz-align-self` and `-moz-overflow-scrolling`. I probably should place an example of vendor-prefixed properties that are not considered warnings.